ape_alchemy/provider.py

```diff
@@ -179,26 +236,60 @@ def get_virtual_machine_error(self, exception: Exception, **kwargs) -> VirtualMa
 
         return VirtualMachineError(message=message, txn=txn)
 
-    def _make_request(self, endpoint: str, parameters: Optional[List] = None) -> Any:
-        try:
-            return super()._make_request(endpoint, parameters)
-        except HTTPError as err:
-            response_data = err.response.json() if err.response else {}
-            if "error" not in response_data:
-                raise AlchemyProviderError(str(err)) from err
-
-            error_data = response_data["error"]
-            message = (
-                error_data.get("message", str(error_data))
-                if isinstance(error_data, dict)
-                else error_data
-            )
-            cls = (
-                AlchemyFeatureNotAvailable
-                if "is not available" in message
-                else AlchemyProviderError
-            )
-            raise cls(message) from err
+    def _make_request(
```

Comment:
The base class has this same, internal method with different kwargs now, I wonder if that will cause problems?

Reply:
if they're strictly after the existing (unchanged) signature, then I think it's probably fine? since they're strictly optional. baking them into the parameters parameter would clean this up, but I have no idea where that's created or what it's used for.